Big-practice/src/script/app.js

```diff
@@ -0,0 +1,197 @@
+class getData {
+    constructor(api) {
+        this.api = api;
+    }
+    fetchData() {
+        fetch(this.api)
+            .then(res => {
+                return res.json();
+            })
+            .then(data => {
+                data.map(element => {
+                    //console.log(element);
+                    todos.push(element);
+                })
+                localStorage.setItem('todo-list', JSON.stringify(todos));
+                showTodo(document.querySelector("span.selected").id);
+            })
+            .catch(error => {
+                console.log(error);
+            })
+    }
+}
```

Comment:
@Quocs Is it correct? Why are you creating a `getData` class instead of a `ToDo` class?

Reply:
Just Change Name Class getData to toDo

Reply:
@Quocs Wew, please help re-read Classess part in JS. What are properties of the ToDo class?